### DISCUSSION TITLE
duda sobre el codigo de honor

### DISCUSSION THREAD

**Discussion by llambias** [IIC2133-PUC/2024-1#127] [Ayuda] [answered]
Hola tengo una duda sobre el codigo de honor
este dice "Tambien se entiende por copia extraer contenido sin modificarlo sustancialmente desde fuentes digitales
como Wikipedia o mediante el uso de asistentes inteligentes como ChatGPT o Copilot", 
quería saber para el uso de inteligencia artificial como herramienta, asumiendo que se cumplan con "modificarlo sustancialmente",¿ este debe ser anexado y declarado en el informe o dentro del mismo codigo para indicar las partes en las que se utilizo? sino ¿cual es el metodo correcto?
gracias

**Comment by Bjnazar** [comment, marked as answer]
Hola!
En el archivo README.md debes mencionar explícitamente qué usaste y donde lo usaste (en qué archivo y en qué parte del código). Si no existe el archivo, lo debes crear.
En el código mismo también debes poner un comentario donde corresponda.
Además, es muy importante que no sea "copy paste" literalmente, sino que entiendas lo que usaste y lo adaptes tú mismo a la tarea en específico.
**Para la realización del informe no se permite el uso de IAs.**
